src/Components/HelperText/HelperText.ts

```diff
@@ -0,0 +1,7 @@
+export const HelperText = (message: string) => {
```

Comment:
This is not a component, so the first letter should not be capitalized
```suggestion
export const helperText = (message: string) => {
```